actions/databases.go

```diff
@@ -0,0 +1,251 @@
+package actions
+
+import (
+	"errors"
+	"log"
+	"strconv"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/gobuffalo/buffalo"
+)
+
+// DatabasesGet gets a list of databases for a given account
+//   If a {db} param is given it will return information about the specific database instance, otherwise
+//   all database instances will be returned.
+//   If the `all=true` parameter is passed it will return a list of clusters in addition to instances.
+func DatabasesGet(c buffalo.Context) error {
+	// if all param is given, we'll return information about both instances and clusters
+	// otherwise, only database instances will be searched
+	all := false
+	if b, err := strconv.ParseBool(c.Param("all")); err == nil {
+		all = b
+	}
+
+	rdsClient := RDS[c.Param("account")]
+	var outputClusters *rds.DescribeDBClustersOutput
+	var outputInstances *rds.DescribeDBInstancesOutput
+	var err error
+	clusterNotFound := true
+
+	if all {
+		clusterNotFound = false
+		inputClusters := &rds.DescribeDBClustersInput{
+			DBClusterIdentifier: aws.String(c.Param("db")),
```

Comment:
If we split out the list and the show, can we avoid this weird logic with `clusterNotFound`?

Reply:
Yeah, I thought about that, will probably make the whole thing simpler.